frontend/src/components/authentication/SignUp.js

```diff
@@ -38,34 +40,42 @@ export default function SignUp() {
 
     axios({
       method: "POST",
-      url:"/register",
-      data:{
+      url: "/register",
+      data: {
         email: signUpForm.email,
         password: signUpForm.password,
         firstName: signUpForm.firstName,
-        lastName: signUpForm.lastName      }
-    })
-    .then((response) => {
-      console.log(response)
-      if (response.status === 200 & response.data.msg === "register successful") {
-        console.log("User created")
-        history.push ("/");
+        lastName: signUpForm.lastName
       }
-    }).catch((error) => {
-      if (error.response) {
-        console.log(error.response)
-        console.log(error.response.status)
-        console.log(error.response.headers)
-        }
     })
+      .then((response) => {
+        console.log(response)
+        if (response.status === 200 & response.data.msg === "register successful") {
+          console.log("User created")
+          history.push("/");
+          setUserExists("Successfully registered and user created, sign in now.")
+        }
+        else if (response.data.msg === "User already exists") {
+          console.log("User already exists");
+          setUserExists("User already exists! Please sign in using your existing credentials.")
```

Comment:
Can we show error message in red and success message in green. 
And have it centered under the form